statsmodels/genmod/families/family.py

```diff
@@ -72,7 +75,14 @@ def _getlink(self):
     link = property(_getlink, _setlink, doc="Link function for family")
 
     def __init__(self, link, variance):
-        self.link = link()
+        if inspect.isclass(link):
```

Comment:
It seems that `inspect.isclass(object)` is implemented as `isinstance(object, type)`, see [source](https://hg.python.org/cpython/file/tip/Lib/inspect.py). But this might work for python 3 only. According to [this stackoverflow answer](http://stackoverflow.com/a/10123520), the python 2 version needs `types.ClassType` which was removed in python 3. So I would stick with `inspect.isclass`.